packages/color-slider/test/color-slider.test.ts

```diff
@@ -215,7 +216,7 @@ describe('ColorSlider', () => {
 
         await elementUpdated(el);
 
-        expect(el.sliderHandlePosition).to.equal(2);
+        expect(el.sliderHandlePosition).to.equal((2 * 100) / 360);
```

Comment:
Can we match these tests with `expect(el.value)` tests so they are easier to read now that the step is more sensitive to the actual values we're managing?